### PR TITLE
Get literal info

### PR DESCRIPTION
**Pull request by ptorrestr**
Hi,

This patch provides access to functions `getNobjectsLiterals()` and `getNobjectsNotLiterals()` in `Dictionary` interface. This is very hand when one wants to know if a certain object is a literal or not.

Also, it fixes some bugs regarding the reading of properties in the initialisation of `BasicHDT` and `FourSectionDictionary`.

Finally, it solves the issue of compiling tests without C++11 support (#168). 

Two tests were included for both changes.

### CODE REVIEW

**Comment by dinikolop** [comment]
Hi @ptorrestr! Sounds great, I will have a look on the changes. A first question:
If I am not mistaken you have already fixed the issue with C++11 support in #169, haven't you? Is there something else related to this issue here?

**Comment by ptorrestr** [comment]
Hi @dinikolop,

You are totally right, I forgot that this was already fixed in #169 

Cheers

File: libhdt/src/dictionary/FourSectionDictionary.cpp [hunk 1]
```diff
@@ -58,7 +58,7 @@ FourSectionDictionary::FourSectionDictionary(HDTSpecification & spec) : blocksiz
 
 	string blockSizeStr = "";
 	try{
-		spec.get("dict.block.size");
+		blockSizeStr = spec.get("dict.block.size");
```
[the remark this answers is not in the document]
Comment (ptorrestr) [reply libhdt/src/dictionary/FourSectionDictionary.cpp]: I'm parsing now the value. I think should be possible to set the bucket size. However, other people might have different thoughts. If that is the case, we should remove this option.

I don't like to include validations regarding the configuration file at this level of the code. I include some here but this should be moved to some other place. In fact, the whole configuration should be refactored and properly documented.  I might work on this later on.

[the remark this answers is not in the document]
Comment (ptorrestr) [reply libhdt/src/dictionary/FourSectionDictionary.cpp]: Yes, that should be the way. We can integrate this PR and start a new PR for refactoring the validation of configuration files

**Comment by LaurensRietveld** [comment]
Great, merged :+1: . Thanks @ptorrestr 

@webdata @RubenVerborgh , mind adding @dinikolop as a committer to this repo? (he's doing most of our hdt commits now)

**Pull request merged**